runtime/runtime/Cargo.toml

```diff
@@ -13,7 +13,7 @@ serde_json = "1.0"
 log = "0.4"
 tracing = "0.1"
 rand = "0.7"
-lazy_static = "1.4"
+once_cell = "1.5.2"
```

Comment:
out of curiosity - why did we update ? (just to have the newest version or was there something wrong with the old one?)

Reply:
@mm-near No, we are still using the old library. The newest version is 1.8.0